packages/babel-runtime/package.json

```diff
@@ -26,6 +26,15 @@
       "./helpers/AsyncGenerator.js"
     ],
     "./helpers/esm/AsyncGenerator": "./helpers/esm/AsyncGenerator.js",
+    "./helpers/AwaitValue": [
```

Comment:
These package.json files are now just sorted alphabetically, but the contents are the same.